### ISSUE TITLE
index route in sitemap

### ISSUE THREAD

**Issue by bux** [jbaubree/vite-ssg-sitemap#2]
vite-ssg-sitemap: v0.1.4

The sitemap generated contains the index routes with `/index` as part of the url. Usually one would just omit `/index`.

```xml
<url>
    <loc>http://localhost/index</loc>
    <lastmod>2022-03-07T05:57:57.333Z</lastmod>
    <changefreq>daily</changefreq>
    <priority>1.0</priority>
</url>
```

I couldn't find an option to adjust this, am I missing something?

**Comment by jbaubree** [comment]
Can you share me your ssgOptions?

**Comment by bux** [comment]
I tested this in vitesse.
Just cloned it after your PR was merged.

**Comment by jbaubree** [comment]
Thanks for this catch, i have just fixed it on [742d4f5](https://github.com/jbaubree/vite-ssg-sitemap/commit/742d4f5a6760d64fec44ebd9e7dac4b27099e992), you can update vite-ssg-sitemap to [v0.1.5](https://github.com/jbaubree/vite-ssg-sitemap/commit/742d4f5a6760d64fec44ebd9e7dac4b27099e992) :)

**Comment by bux** [comment]
This has fixed the issue. Thanks :)
